productivity/content.css

```
@@ -57,13 +56,10 @@
 .ck-content ol,
 .ck-content p,
 .ck-content ul {
-  font-size: 1em;
-  line-height: 1.6em;
   padding-top: 0.2em;
   margin-bottom: var(--ck-spacing-large);
 }
 
 .ck-content {
   font-family: "Times New Roman", Time, serif;
```

To make it really work we must set font-family as a variable.

```suggestion
  --ck-content-font-family: "Times New Roman", Time, serif;
```